mopidy/core/mixer.py

```diff
@@ -99,3 +100,14 @@ def set_mute(self, mute):
             return result
 
         return False
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return MixerState(volume=self.get_volume())
+
+    def _restore_state(self, state, coverage):
+        """Internal method for :class:`mopidy.Core`."""
+        if state:
+            if 'volume' in coverage:
+                if state.volume:
```

Comment:
Hmm. If you don't insist, I keep it as it is. Then all '_restore_state' function look alike. See playback.py and tracklist.py where I can't optimize that. Also it highlights that `state` can be `None` (and will not be ignored for future extension).